### PR TITLE
skeleton for concentration profile plot

### PR DESCRIPTION
**Pull request by IndrajeetPatil**

### CODE REVIEW

File: appveyor.yml [hunk 1]
```diff
@@ -39,9 +39,11 @@ before_build:
 
 build_script:
   - Rscript -e "install.packages('https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1/rClr_0.9.1.zip', repos = NULL, type = 'binary')"
+  - Rscript -e "install.packages('https://ci.appveyor.com/api/buildjobs/s4nhbp52qrknf40m/artifacts/tlf_1.2.43.zip', repos = NULL, type = 'binary')"
```
Comment (Yuri05) [inline appveyor.yml:42]: if you want to automatically use the latest available version of tlf on AppVeyor, you could use the permalink like e.g. in the Reporting Engine:
https://github.com/Open-Systems-Pharmacology/OSPSuite.ReportingEngine/blob/dbd1435fec9bee4559dad8695484fd2d520457f7/appveyor.yml#L38

File: appveyor.yml [hunk 2]
```diff
@@ -39,9 +39,11 @@ before_build:
 
 build_script:
   - Rscript -e "install.packages('https://github.com/Open-Systems-Pharmacology/rClr/releases/download/v0.9.1/rClr_0.9.1.zip', repos = NULL, type = 'binary')"
+  - Rscript -e "install.packages('https://ci.appveyor.com/api/buildjobs/s4nhbp52qrknf40m/artifacts/tlf_1.2.43.zip', repos = NULL, type = 'binary')"
   - travis-tool.sh install_deps
   - travis-tool.sh r_binary_install curl
   - Rscript -e "install.packages('covr', repos = 'http://cran.us.r-project.org')"
+  - Rscript -e "install.packages(c('ggplot2', 'jsonlite', 'reshape2'), repos = 'http://cran.us.r-project.org')"
```
Comment (Yuri05) [inline appveyor.yml:46]: any reason why the covr package above is installed separately and not just included into the packages list here?

File: R/plotIndividualTimeProfile.R [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+#'   - `"sqrt"` (for continusous square root scale)
+#' @param title,subtitle,xlabel,ylabel,legendTitle A character/string defining
+#'   plot title, subtitle, axes labels, legend title, respectively.
+#'
+#' @import tlf
+#'
+#' @export
+plotIndividualTimeProfile <- function(dataCombined,
+                                      xAxisScale = "lin",
+                                      yAxisScale = "lin",
```
Comment (Yuri05) [inline R/plotIndividualTimeProfile.R:21]: shouldn't we use `Scaling$lin` from the tlf enum instead?

**Comment by IndrajeetPatil** [comment]
This is still a draft PR. Please don't review this yet. I will mark it ready for review soon.

Do discuss this here though: https://github.com/Open-Systems-Pharmacology/OSPSuite-R/discussions/833

**Comment by Yuri05** [comment]
ok, no problem :)

**Comment by IndrajeetPatil** [comment]
Closing in favour of #858. It will be part of that PR.

**Pull request closed without merging**